buildspec.yml

```diff
@@ -0,0 +1,22 @@
+version: 0.2         
+phases:
+  install:
+    runtime-versions:
+      golang: 1.12
+  pre_build:
+    commands:
+      - echo Building the AWS for Fluent Bit image
+  build:
+    commands:
+      # Command to build your project
+      - make release
+
+      # List the docker images 
+      - docker images
```

Comment:
why list the images?

Reply:
If anything bad happens, it will help us to debug seeing the logs in our codebuild project. I put it intentionally to see what images we are getting after the build.